diff --git a/filezilla/src/interface/Options.cpp b/filezilla/src/interface/Options.cpp
index b48063e..9933e59 100644
--- a/filezilla/src/interface/Options.cpp
+++ b/filezilla/src/interface/Options.cpp
@@ -42,10 +42,10 @@ static unsigned int register_interface_options()
 		{ "Cache directory", L"", option_flags::predefined_priority | option_flags::platform },
 
 		// Normal UI options
-		{ "Number of Transfers", 2, option_flags::numeric_clamp, 1, 10 },
+		{ "Number of Transfers", 2, option_flags::numeric_clamp, 1, 50 },
 		{ "Language Code", L"", option_flags::normal, 50 },
-		{ "Concurrent download limit", 0, option_flags::numeric_clamp, 0, 10 },
-		{ "Concurrent upload limit", 0, option_flags::numeric_clamp, 0, 10 },
+		{ "Concurrent download limit", 0, option_flags::numeric_clamp, 0, 50 },
+		{ "Concurrent upload limit", 0, option_flags::numeric_clamp, 0, 50 },
 		{ "Show debug menu", false, option_flags::normal },
 		{ "File exists action download", 0, option_flags::normal, 0, 7 },
 		{ "File exists action upload", 0, option_flags::normal, 0, 7 },
diff --git a/filezilla/src/interface/settings/optionspage_transfer.cpp b/filezilla/src/interface/settings/optionspage_transfer.cpp
index 6b5f223..0256665 100644
--- a/filezilla/src/interface/settings/optionspage_transfer.cpp
+++ b/filezilla/src/interface/settings/optionspage_transfer.cpp
@@ -49,19 +49,19 @@ bool COptionsPageTransfer::CreateControls(wxWindow* parent)
 		auto [box, inner] = lay.createStatBox(main, _("Concurrent transfers"), 3);
 		inner->Add(new wxStaticText(box, nullID, _("Maximum simultaneous &transfers:")), lay.valign);
 		impl_->transfers_ = new wxSpinCtrlEx(box, nullID, wxString(), wxDefaultPosition, wxSize(lay.dlgUnits(26), -1));
-		impl_->transfers_->SetRange(1, 10);
+		impl_->transfers_->SetRange(1, 50);
 		impl_->transfers_->SetMaxLength(2);
 		inner->Add(impl_->transfers_, lay.valign);
-		inner->Add(new wxStaticText(box, nullID, _("(1-10)")), lay.valign);
+		inner->Add(new wxStaticText(box, nullID, _("(1-50)")), lay.valign);
 		inner->Add(new wxStaticText(box, nullID, _("Limit for concurrent &downloads:")), lay.valign);
 		impl_->downloads_ = new wxSpinCtrlEx(box, nullID, wxString(), wxDefaultPosition, wxSize(lay.dlgUnits(26), -1));
-		impl_->downloads_->SetRange(0, 10);
+		impl_->downloads_->SetRange(0, 50);
 		impl_->downloads_->SetMaxLength(2);
 		inner->Add(impl_->downloads_, lay.valign);
 		inner->Add(new wxStaticText(box, nullID, _("(0 for no limit)")), lay.valign);
 		inner->Add(new wxStaticText(box, nullID, _("Limit for concurrent &uploads:")), lay.valign);
 		impl_->uploads_ = new wxSpinCtrlEx(box, nullID, wxString(), wxDefaultPosition, wxSize(lay.dlgUnits(26), -1));
-		impl_->uploads_->SetRange(0, 10);
+		impl_->uploads_->SetRange(0, 50);
 		impl_->uploads_->SetMaxLength(2);
 		inner->Add(impl_->uploads_, lay.valign);
 		inner->Add(new wxStaticText(box, nullID, _("(0 for no limit)")), lay.valign);
@@ -184,16 +184,16 @@ bool COptionsPageTransfer::SavePage()
 
 bool COptionsPageTransfer::Validate()
 {
-	if (impl_->transfers_->GetValue() < 1 || impl_->transfers_->GetValue() > 10) {
-		return DisplayError(impl_->transfers_, _("Please enter a number between 1 and 10 for the number of concurrent transfers."));
+	if (impl_->transfers_->GetValue() < 1 || impl_->transfers_->GetValue() > 50) {
+		return DisplayError(impl_->transfers_, _("Please enter a number between 1 and 50 for the number of concurrent transfers."));
 	}
 
-	if (impl_->downloads_->GetValue() < 0 || impl_->downloads_->GetValue() > 10) {
-		return DisplayError(impl_->downloads_, _("Please enter a number between 0 and 10 for the number of concurrent downloads."));
+	if (impl_->downloads_->GetValue() < 0 || impl_->downloads_->GetValue() > 50) {
+		return DisplayError(impl_->downloads_, _("Please enter a number between 0 and 50 for the number of concurrent downloads."));
 	}
 
-	if (impl_->uploads_->GetValue() < 0 || impl_->uploads_->GetValue() > 10) {
-		return DisplayError(impl_->uploads_, _("Please enter a number between 0 and 10 for the number of concurrent uploads."));
+	if (impl_->uploads_->GetValue() < 0 || impl_->uploads_->GetValue() > 50) {
+		return DisplayError(impl_->uploads_, _("Please enter a number between 0 and 50 for the number of concurrent uploads."));
 	}
 
 	if (fz::to_integral<int>(impl_->dllimit_->GetValue().ToStdWstring(), -1) < 0) {
diff --git a/filezilla/src/interface/sitemanager_controls.cpp b/filezilla/src/interface/sitemanager_controls.cpp
index a8b254b..1217c16 100644
--- a/filezilla/src/interface/sitemanager_controls.cpp
+++ b/filezilla/src/interface/sitemanager_controls.cpp
@@ -1319,7 +1319,7 @@ TransferSettingsSiteControls::TransferSettingsSiteControls(wxWindow & parent, Di
 	row->Add(new wxStaticText(&parent, nullID, _("&Maximum number of connections:")), lay.valign);
 	impl_->max_conns_ = new wxSpinCtrlEx(&parent, nullID, wxString(), wxDefaultPosition, wxSize(lay.dlgUnits(26), -1));
 	impl_->max_conns_->SetMaxLength(2);
-	impl_->max_conns_->SetRange(1, 10);
+	impl_->max_conns_->SetRange(1, 50);
 	row->Add(impl_->max_conns_, lay.valign);
 
 	impl_->limit_max_conns_->Bind(wxEVT_CHECKBOX, [&](wxCommandEvent const& ev){ impl_->max_conns_->Enable(ev.IsChecked()); });
